test/arrays/ferguson/semantic-examples/0-init-alias.chpl

```diff
@@ -0,0 +1,10 @@
+use samples;
+// var one = new R(1);
```

Comment:
My point is that they look like you've commented out code, but then it becomes confusing when you see that the variable is still used.  I'd prefer they get at least changed so that it's clear the code is from another file that you've included to make the remainder more readable